VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts

```diff
@@ -45,33 +49,66 @@ export class JudgeParticipantStatusListComponent implements OnInit {
 
     private camelToSpaced(word: string) {
         return word.split(/(?=[A-Z])/).join(' ');
-    }
+  }
 
-    getParticipantStatusCss(participant: ParticipantResponse): string {
+  getParticipantStatusCss(participant: ParticipantResponse): string {
         if (participant.status === ParticipantStatus.None) {
             return this.camelToSnake(ParticipantStatus.NotSignedIn.toString());
         } else {
             return this.camelToSnake(participant.status.toString());
-        }
+    }
     }
 
     private camelToSnake(word: string) {
         return word
             .split(/(?=[A-Z])/)
             .join('_')
             .toLowerCase();
-    }
+  }
 
-    isUserJudge(): boolean {
-        const participant = this.conference.participants.find(
-            x => x.username.toLowerCase() === this.adalService.userInfo.userName.toLocaleLowerCase()
-        );
-        return participant.role === Role.Judge;
-    }
+  isUserJudge(): boolean {
+    const participant = this.conference.participants.find(
+      x => x.username.toLowerCase() === this.adalService.userInfo.userName.toLocaleLowerCase()
+    );
+    return participant.role === Role.Judge;
+  }
 
-    private filterRepresentatives(): void {
-        this.representativeParticipants = this.conference.participants.filter(x => x.role === Role.Representative);
-        this.litigantInPerson = this.representativeParticipants.length === 0;
-        this.individualParticipants = this.conference.participants.filter(x => x.role === Role.Individual);
-    }
+  private filterRepresentatives(): void {
+    this.representativeParticipants = this.conference.participants.filter(x => x.role === Role.Representative);
+    this.litigantInPerson = this.representativeParticipants.length === 0;
+    this.individualParticipants = this.conference.participants.filter(x => x.role === Role.Individual);
+  }
+
+  changeJudgeNameShow() {
+    this.showChangeJudgeDisplayName = true;
+    this.newJudgeDisplayName = this.judge.display_name;
+  }
+
+  onEnterJudgeDisplayName(value: string) {
+    this.newJudgeDisplayName = value;
+  }
+
+  saveJudgeDisplayName() {
+    this.judge.display_name = this.newJudgeDisplayName;
+    this.showChangeJudgeDisplayName = false;
+    this.updateParticipant();
+  }
+
+  cancelJudgeDisplayName() {
+    this.showChangeJudgeDisplayName = false;
+  }
+
+  private updateParticipant() {
+    const updateParticipantRequest = new UpdateParticipantRequest(
+      {
+        fullname: this.judge.name,
+        display_name: this.judge.display_name,
+        representee: this.judge.representee
+      });
+
+    this.videoWebService.updateParticipantBookingDetails(this.conference.id, this.judge.id, updateParticipantRequest)
```

Comment:
Hi Valentina, Can you confirm what you are trying to do here. The name "updateParticipantBookingDetails"  suggest that you are trying to update the booking database. But the code seems to call the Video API 
 let url_ = this.baseUrl + "/conferences/{conferenceId}/participants/{participantId}/participantDisplayName";

Can you confirm and rename based on above?

Reply:
I renamed it , removed 'booking', it 's call video api endpoint to update display name in video database.